packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -102,12 +104,25 @@ def config_all_loggers(*, log_format_local_dev_enabled: bool) -> None:
         fmt = LOCAL_FORMATTING
 
     for logger in loggers:
-        set_logging_handler(
+        _set_logging_handler(
             logger, fmt=fmt, log_format_local_dev_enabled=log_format_local_dev_enabled
         )
 
+    for logger_name, filtered_routes in logger_filter_mapping.items():
+        logger = logging.getLogger(logger_name)
+        # Check if the logger has any handlers or is in active use
+        if not logger.hasHandlers():
+            _logger.warning(
+                f"Logger %s does not have any handlers. Filter will not be added.",
+                logger_name,
+            )
+            continue
```

Comment:
is this an unexpected condition? if yes I think this should raise not continue silently. Otherwise no one will ever find it out

Reply:
at least an assert would be in order so that this would be catched in tests

Reply:
No it is expected, as `servicelib` is used across all services and some let say 3rd party loggers might not be used in some of those services.